example/k3d/smoke/main.jsonnet

```diff
@@ -34,11 +34,11 @@ local smoke = {
   avalanche: avalanche.new(replicas=3, namespace='smoke', config={
     // We're going to be running a lot of these and we're not trying to test
     // for load, so reduce the cardinality and churn rate.
-      avalanche: avalanche.new(replicas=3, namespace='smoke', config={
-        metric_count: 1000,
-        series_interval: 300,
-        metric_interval: 600,
-      }),
+    avalanche: avalanche.new(replicas=3, namespace='smoke', config={
+      metric_count: 1000,
+      series_interval: 300,
+      metric_interval: 600,
+    }),
   }),
```

Comment:
Sorry, you're getting a nit since this got auto formatted :) 

This has actually been broken for a while and a PR hasn't fixed it yet. This is accidentally nesting a second call to avalanche.new when it shouldn't be, instead just giving the config settings. Can you fix that?

Reply:
Done! Fixed formatting in a couple other places too 35cdcaecad7e0aceaf3bc399af4da8fd4818ebaa